src/gui/rss/rsswidget.h

```diff
@@ -24,9 +26,8 @@
  * modify file(s), you may extend this exception to your version of the file(s),
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
- *
- * Contact : chris@qbittorrent.org arnaud@qbittorrent.org
  */
+
 #ifndef __RSS_IMP_H__
 #define __RSS_IMP_H__
```

Comment:
Will be fixed.

>Also i noticed you use `#pragma once` in half the header files but include guards in the other half.

Almost all the files under *base/rss* have been heavily rewritten (or rather, they were created from scratch using the old code) and I decided to use `#pragma once` there. My goal was not to change it everywhere.